runtime/onert/backend/cpu/ops/RoPELayer.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_BACKEND_CPU_OPS_ROPE_LAYER_H__
+#define __ONERT_BACKEND_CPU_OPS_ROPE_LAYER_H__
+
+#include <backend/IPortableTensor.h>
+#include "OperationUtils.h"
+
+#include <exec/IFunction.h>
+
+namespace nnfw
+{
+namespace cker
+{
+class RoPE;
```

Comment:
It looks like Forward declarations. But I couldn't find the `RoPE` class elsewhere. 
May I ask why this `RoPE` class was declared here? 

```suggestion
```

Reply:
I added it because I thought forward degradation was necessary when implementing draft, but looking at the current code, I think It doesn't seem necessary also.
i will delete it. thanks